zerver/tests/test_events.py

```diff
@@ -2194,7 +2194,7 @@ def test_change_bot_owner(self) -> None:
         self.user_profile = self.example_user('hamlet')
         bot = self.create_test_bot('test2', previous_owner, full_name='Test2 Testerson')
         action = lambda: do_change_bot_owner(bot, self.user_profile, previous_owner)
-        events = self.do_test(action)
+        events = self.do_test(action, num_events=2)
```

Comment:
We should add a schema checker for the second event in these tests, similar to `change_bot_owner_checker`

Reply:
Done!